projects/is-cron-editor/src/lib/is-cron-editor.component.ts

```diff
@@ -1003,7 +1007,9 @@ export class IsCronEditorComponent implements OnInit, ControlValueAccessor, Vali
     if (this._changeSubscription) {
       this._changeSubscription.unsubscribe();
     }
-    this._changeSubscription = this.cronExpressionControl.valueChanges.subscribe(fn);
+    this._changeSubscription = this.cronExpressionControl.valueChanges.pipe(
```

Comment:
Please rewrite `registerOnChange` similar way to `registerOnTouched` and call stored function (if defined) within previous subscription to `cronExpressionControl`